community/examples/htc-slurm.yaml

```diff
@@ -69,14 +69,6 @@ deployment_groups:
       size_gb: 10240
       local_mount: /projects
 
-  # This file system has an associated license cost.
-  # https://console.developers.google.com/marketplace/product/ddnstorage/exascaler-cloud
-  - id: scratchfs
-    source: community/modules/file-system/DDN-EXAScaler
-    use: [network]
-    settings:
-      local_mount: /scratch
-
```

Comment:
Here I am wondering if we should review this blueprint. I think we should perhaps simplify everything and only have a managed lustre, rather than 2 different tiers of filestore and one lustre. 
@samskillman do you have an opinion?

Reply:
I discussed with Sam, I would propose that you keep only the home file systems and remove the others.

Reply:
could you please also remove projects fs, and adjust this blueprint documentation in examples/README if needed?